mixpanel/__init__.py

```diff
@@ -384,11 +384,11 @@ def __init__(self, max_size=50, events_url=None, people_url=None, import_url=Non
         self._buffers = {
             'events': [],
             'people': [],
-            'imports': [],
+            'imports': {},
```

Comment:
Discussed, we could bring this out into `self._imports` since we're special casing it (and lookups that result in `'imports'` will always be an error